cf-nodejs-logging-support-core/log-core.js

```diff
@@ -292,9 +294,15 @@ var initLog = function () {
     var logObject = JSON.parse(initDummy);
 
     var now = new Date();
-
     logObject.written_at = now.toJSON();
-    logObject.written_ts = now.getTime() * NS_PER_MS + (process.hrtime()[1] % 1000000);
+
+    var lower = process.hrtime()[1] % 1000000
+    var higher = now.getTime() * NS_PER_MS
+
+    logObject.written_ts = higher + lower;
+    if(logObject.written_ts < lastTimestamp)
```

Comment:
MAybe add a comment, why this could happen.

Reply:
Added comment on why this is needed